StereoKitC/platforms/linux.cpp

```diff
@@ -438,6 +437,10 @@ bool linux_start_post_xr() {
 
 bool linux_start_flat() {
 	#if defined(SKG_LINUX_EGL)
+	if (skg_egl_dri()) {
```

Comment:
If `setup_x_window` will fail here anyhow, I think it'd be fine to let that one do its thing and error out, instead of checking on dri? I think the error there will be plenty clear.